app/src/main/java/com/hedvig/app/feature/home/ui/HomeFragment.kt

```diff
@@ -113,12 +115,10 @@ class HomeFragment : Fragment(R.layout.home_fragment) {
                 return@observe
             }
             if (isPending(successData.contracts)) {
-                val items = mutableListOf<HomeModel>().apply {
+                val items = buildList {
```

Comment:
A bit of a nicer way to build those lists. I know we have to add the optIn for the experimental stdlib api but they will be stable in Kotlin 1.6 https://kotlinlang.org/docs/whatsnew16.html#stable-collection-builders so there's no risk of needing to migrate off of it anymore so we can use it.
Benefits mainly come from it returning the list as a non-mutable one outside of the lambda, so no way to accidentally mutate it when you shouldn't.

Reply:
Nice